go/otgrpc/server.go

```diff
@@ -64,7 +64,7 @@ func OpenTracingServerInterceptor(tracer opentracing.Tracer, optFuncs ...Option)
 				serverSpan.LogFields(log.Object("gRPC response", resp))
 			}
 		} else {
-			ext.Error.Set(serverSpan, true)
+			SetSpanTags(serverSpan, err, false)
 			serverSpan.LogFields(log.String("event", "gRPC error"), log.Error(err))
```

Comment:
This can technically be moved into `SetSpanTags` too, with the method renamed to something like `LogErrorSpan`, I left it as-is for the time being as I don't have strong opinions.